### PR TITLE
Update R&D organization and AORs

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Updated to match the new org structure and AORs.

### CODE REVIEW

**Review by justinegeffen — DISMISSED** [review]
Thanks for this update!

**Review by kwiersgalla — DISMISSED** [review]
Thanks Joram.

File: operations/research-and-development/organization/README.md [hunk 1]
```diff
@@ -1,51 +1,54 @@
...
-* **[SRE](/operations/research-and-development/organization/sre.md)**
+* **[Site Reliabilty Engineering](/operations/research-and-development/organization/sre.md)**
 
 ## Security
 
-The Security division is responsible for both product and security operations.
+The Security division is responsible for both product and security operations, in addition to security certifications.
 
-* **[Product Security](/operations/research-and-development/organization/product_security.md)** 
+* **[Product Security](/operations/research-and-development/organization/product_security.md)**
```
Comment (DSchalla) [inline operations/research-and-development/organization/README.md:54]:
```suggestion
* **[Governance, Risk, and Compliance (GRC)](/operations/research-and-development/organization/grc.md)**
* **[Product Security](/operations/research-and-development/organization/product_security.md)**
```

File: operations/research-and-development/organization/README.md [hunk 2]
```diff
@@ -1,51 +1,54 @@
...
+
 * **[Release DevOps](/operations/research-and-development/organization/release_devops.md)**
 * **[Cloud Platform](/operations/research-and-development/organization/cloud_platform.md)**
-* **[SRE](/operations/research-and-development/organization/sre.md)**
+* **[Site Reliabilty Engineering](/operations/research-and-development/organization/sre.md)**
 
 ## Security
 
-The Security division is responsible for both product and security operations.
+The Security division is responsible for both product and security operations, in addition to security certifications.
```
Comment (DSchalla) [inline operations/research-and-development/organization/README.md]:
```suggestion
The Security division is responsible for the implementation and monitoring of the company's security program.
```

File: SUMMARY.md [hunk 3]
```diff
@@ -45,23 +45,19 @@
...
     * [Tech Writing](/operations/research-and-development/organization/tech_writing.md)
     * [Growth](/operations/research-and-development/organization/growth.md)
-    * [Self-serve subscriptions](/operations/research-and-development/organization/self-serve_subscriptions.md)
+    * [Self-serve](/operations/research-and-development/organization/self-serve_subscriptions.md)
     * [Data engineering](/operations/research-and-development/organization/data_engineering.md)
     * [Release DevOps](/operations/research-and-development/organization/release_devops.md)
     * [Cloud Platform](/operations/research-and-development/organization/cloud_platform.md)
-    * [SRE](/operations/research-and-development/organization/sre.md)
+    * [Site Reliability Engineering](/operations/research-and-development/organization/sre.md)
     * [Product Security](/operations/research-and-development/organization/product_security.md)
```
Comment (DSchalla) [inline SUMMARY.md:62]:
```suggestion
    * [GRC](/operations/research-and-development/organization/grc.md)
    * [Product Security](/operations/research-and-development/organization/product_security.md)
```

**Review by DSchalla — CHANGES_REQUESTED** [review]
Looks good overall, thanks Joram! I've committed a team page for the GRC team to the branch directly and made some minor change requests to add them in all places.

**Comment by DSchalla** [comment]
@justinegeffen @kwiersgalla Apologies, it seems my commit addition invalidated your review.

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
> @justinegeffen @kwiersgalla Apologies, it seems my commit addition invalidated your review.

No worries! I've requested another review from you - I think if you approve it'll mark your requests as done. I never know with GitHub, but your changes have been merged!

**Review by justinegeffen — DISMISSED** [review]

**Review by DSchalla — DISMISSED** [review]
Looks good, thanks @justinegeffen!

**Review by kwiersgalla — DISMISSED** [review]

**Review by marianunez — DISMISSED** [review]
Thanks @jwilander! I made some minor updates on Growth and Self-Serve AORs. My changes dismissed the existing approvals 😬

File: operations/research-and-development/organization/growth.md [hunk 4]
```diff
@@ -5,21 +5,18 @@ The Growth team is responsible for accelerating usage and revenue by optimizing
...
 - User Feedback & Announcements
   - NPS system
   - Feedback widgets
   - In-product announcements
-  - Admin Advisor
-- Expansion
-  - In-product upsells
-- Packaging & Pricing
-- Growth Telemetry
+  - Admin Advisor
```
Comment (catalintomai) [inline operations/research-and-development/organization/growth.md]: Hasn't this been removed?

File: operations/research-and-development/organization/growth.md [hunk 5]
```diff
@@ -5,21 +5,18 @@ The Growth team is responsible for accelerating usage and revenue by optimizing
...
+  - User Invites
   - Signup-to-login, install-to-login
   - User invitation flows
   - Install-to-dau
-  - Free-to-trial
-  - Trial-to-paid
 - Adoption
-  - First-time user onboarding experience
   - Day 1, Day 7, Day 28 retention
   - DAU Nuturing
```
Comment (catalintomai) [inline operations/research-and-development/organization/growth.md]: Nuturing?

File: operations/research-and-development/organization/growth.md (hunk 4 above)
Comment (jwilander) [reply operations/research-and-development/organization/growth.md]: @marianunez ?

File: operations/research-and-development/organization/growth.md (hunk 5 above)
Comment (jwilander) [reply operations/research-and-development/organization/growth.md]: @marianunez ?

Comment (marianunez) [reply operations/research-and-development/organization/growth.md]: Typo, it is `Nurturing`

File: operations/research-and-development/organization/growth.md (hunk 4 above)
Comment (marianunez) [reply operations/research-and-development/organization/growth.md]: Yes, the original Admin Advisor was disabled. The concept of suggesting specific features depending on usage is still discussed but I'll remove this and the new features are considered under upsells that Self-Serve team owns.

**Review by saturninoabril — APPROVED** [review]
Thanks Joram, LGTM for QA AOR.

File: operations/research-and-development/organization/README.md [hunk 6]
```diff
@@ -1,51 +1,55 @@
...
-Each vertical team is responsible for a single product.
+### Core Product
 
-* **[Core 1](/operations/research-and-development/organization/core1.md)**
-* **[Boards](/operations/research-and-development/organization/boards.md)**
-* **[Playbooks](/operations/research-and-development/organization/playbooks.md)**
-* **[Calls](/operations/research-and-development/organization/calls.md)**
-
-### Suite teams
+The Core Product sub-division is responsible for building the features and systems of the core product.
```
Comment (Willyfrog) [inline operations/research-and-development/organization/README.md:13]: Maybe this sounds better once the teams are no longer named Core, but right now it feels like trying to define  a word with itself.
But I don't have a better suggestion for it. :/

**Review by Willyfrog — APPROVED** [review]

File: operations/research-and-development/organization/README.md (hunk 6 above)
Comment (jwilander) [reply operations/research-and-development/organization/README.md:13]: Yeah, I struggled with that a bit. "The core team builds the core" is a little unnecessary but it also felt weird not providing any description.

**Review by m1lt0n — APPROVED** [review]

**Comment by jwilander** [comment]
Going to merge as is in order to get it in. If folks have updates they'd like to make please follow-up with another PR.

**Pull request merged**